Description: Replace two hyphens by minuses as intended.
Forwarded: no
Author: Tanguy Ortolo <tanguy+debian@ortolo.eu>
Forwared: yes
Last-Update: 2011-10-23

Index: itstool/itstool.1.in
===================================================================
--- itstool.orig/itstool.1.in	2011-09-03 19:14:28.268601294 +0200
+++ itstool/itstool.1.in	2011-09-03 19:15:10.804922772 +0200
@@ -16,13 +16,13 @@
 
 To extract messages from XML files \fBFILES\fR and output them to \fBOUT.pot\fR:
 
-\fBitstool -o OUT.pot FILES\fR
+\fBitstool \-o OUT.pot FILES\fR
 
 After merging with existing translations or translating strings, generate an
 MO file with \fBmsgfmt(1)\fR, then output translated files to the directory
 \fBDIR\fR:
 
-\fBitstool -m OUT.mo -o DIR FILES\fR
+\fBitstool \-m OUT.mo \-o DIR FILES\fR
 
 ITS definitions are loaded from the built-in rules, rules embedded in the source
 XML files, files passed with the \fB-i\fR option, and ITS attributes in the source
